contracts/message-based/PriceAware.sol

```diff
@@ -3,49 +3,40 @@
 pragma solidity ^0.8.2;
 
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
 
-contract PriceAware is Ownable {
+abstract contract PriceAware {
   using ECDSA for bytes32;
 
-  uint256 public maxDelay = 3 * 60;
-  address private trustedSigner;
+  uint256 public maxDelay = 3 * 60; // seconds
```

Comment:
If we create an abstract contract, let's also get rid of this variable (read a comment below).

Reply:
Ok